release.sh

```diff
@@ -83,14 +89,19 @@ update_version () {
 	echo -e "\\nWriting version file..."
 	echo "${new_version}" > "${VERSION_FILE}"
 
-	echo -e "\\nCommitting version file..."
+	echo -e "Updating ${CHANGELOG}..."
+	sed -i "s/^## \[Unreleased]$/&\n\n## [${new_version}] - $(date +%Y-%m-%d)/" "${CHANGELOG}"
+
+	echo -e "Committing release files..."
 	git add "${VERSION_FILE}"
```

Comment:
dont we need to add the changelog to the commit?

Reply:
Ah, yep 👍